pkg/generated/restapi/configure_fulcio_server.go

```diff
@@ -195,13 +194,8 @@ func logAndServeError(w http.ResponseWriter, r *http.Request, err error) {
 	// errors should always be in JSON
 	w.Header()["Content-Type"] = []string{"application/json"}
 	if e, ok := err.(goaerrors.Error); ok && e.Code() == http.StatusUnauthorized {
-		issuerURL, parseErr := url.Parse(viper.GetString("oidc-issuer"))
-		if parseErr != nil {
-			log.RequestIDLogger(r).Error(fmt.Errorf("error parsing OIDC provider: %w", parseErr))
-		} else {
-			// this is set directly so the header name is not canonicalized
-			w.Header()["WWW-Authenticate"] = []string{fmt.Sprintf("Bearer realm=\"%v\",scope=\"openid email\"", issuerURL.Host)}
-		}
+		// this is set directly so the header name is not canonicalized
+		w.Header()["WWW-Authenticate"] = []string{"Bearer realm=\"oauth2.sigstore.dev\",scope=\"openid email\""}
```

Comment:
nit: this should probably iterate across all issuers, like this:

```go
var b strings.Builder
for i, iss := range fulcioCfg.OIDCIssuers {
	if i != 0 {
		b.WriteString(", ")
	}
	b.WriteString("Bearer realm=\""+iss.IssuerURL+"\",scope=\"openid email\"")
}
w.Header()["WWW-Authenticate"] = []string{b.String()}
```

note this doesn't create multiple WWW-Authenticate response headers but instead returns 1 with all relevant challenges concatenated per https://datatracker.ietf.org/doc/html/rfc7235#section-4.1

Reply:
nice!

Reply:
had to tweak it a bit since it's a map not a slice...